Server/models/articles.model.js

```diff
@@ -12,4 +12,20 @@ exports.fetchArticleByID = (article_id) => {
         else return data.rows[0];
         
     });
+};
+
+exports.fetchAllArticles = () => {
+
+    const queryString = `SELECT articles.author, title, articles.article_id, topic, articles.created_at, articles.votes, article_img_url, 
+    COUNT(comments.article_id) AS comment_count 
```

Comment:
Just be aware, that your comment_count will come back as a string and not an integer.

Reply:
thanks for the heads up :)